web/src/components/Upstream/components/Type.tsx

```diff
@@ -14,28 +14,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import React, { useState } from 'react'
-import { AutoComplete, Form, Select } from 'antd'
-import { useIntl } from 'umi'
-import type { FormInstance } from 'antd/es/form'
+import React, { useState } from 'react';
+import { AutoComplete, Form, Select } from 'antd';
+import { useIntl } from 'umi';
+import type { FormInstance } from 'antd/es/form';
 
-import { HashOnEnum, CommonHashKeyEnum, AlgorithmEnum } from '../constant'
+import { HashOnEnum, CommonHashKeyEnum, AlgorithmEnum } from '../constant';
```

Comment:
I would suggest linting all files in one PR, then submit your changes for this plugin.

Reply:
I will try

Reply:
#2035  @juzhiyuan